### ISSUE TITLE
not an issue - qtn regarding third-party css assets

### ISSUE THREAD

**Issue by alvipeo** [AngularClass/angular-starter#668]
If I include in a prj some third-party lib which has its own css, what's the best way of including it into the prj?

I think of somehow automatically copying it to assets folder. Or, better yet to _require_ it and let webpack to bundle it? How to do it with webpack?

**Comment by loicsalou** [comment]
hi,

I'm very interested in this topic. Any progress ?
I'm currently trying to write my own components and publish to NPM. As long as my components only include typescript code and HTML templates it's working fine. 
But when I need my components to include a css (or scss) file by means of "styleUrls" attribute in @component decorator then it's not working. Same thing if I have, in my component, some png which I want to use either in HTML or css.
I checked that everything is correctly published in my component but starter doesn't seem to care about it.

Anyone tried this (and succeeded ;-)) ? any help, pointer to some similar issue would be appreciated ! I can provide the component I wrote for testing purpose and also my fork of the starter (everything is on github).

Thanks
